.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,18 +12,11 @@
 - [ ] Add pull request hyperlink to relevant TFS items
 - [ ] For large or complex change: schedule an in-person review session
 - [ ] This change has appropriate test coverage
+- [ ] Get at least two approvals
```

Comment:
we can also add all comments/reviews addressed as a final chekpoint

Reply:
That comment goes with all your PRs at this point - don't want to copy paste to all :)